recipes/libtinyiiod/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libtinyiiod" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0b9bc22b2e0090b905fa369fcc652fe3522a68b0aad5ded1d30a1464058596d2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    # verify that headers get installed
+    - test -f $PREFIX/include/tinyiiod/tinyiiod.h  # [not win]
+    - test -f $PREFIX/include/tinyiiod/compat.h  # [not win]
+    - if not exist %PREFIX%\\Library\\include\tinyiiod\tinyiiod.h exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\include\tinyiiod\compat.h exit 1  # [win]
+
+    # verify that libraries get installed
+    - test -f $PREFIX/lib/tinyiiod${SHLIB_EXT}  # [not win]
+    - if not exist %PREFIX%\\Library\\bin\\tinyiiod.dll exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\tinyiiod.lib exit 1  # [win]
```

Comment:
Please add an existence test for one of the cmake files.

Reply:
Updated